examples/misc/lib/effective_dart/usage_bad.dart

```diff
@@ -76,6 +76,26 @@ void miscDeclAnalyzedButNotTested() {
     // #enddocregion use-tear-off
   };
 
+  {
+    // #docregion default-separator
+    void insert(Object item, {int at: 0}) {
+      // ...
```

Comment:
In other code excerpts where there are ellipses we use `/*...*/` in the code:

```dart
void insert(Object item, {int at = 0}) {
  /*...*/
}
```

When this code is extracted into the document, the `/*...*/` is replaced by `...`:

```dart
void insert(Object item, {int at = 0}) {
  ...
}
```

See line 55 above for an example.

Reply:
Done.